File: java/src/main/java/com/dsi/laboreos/model/TipoLaboreo.java
package com.dsi.laboreos.model;

public class TipoLaboreo {
    private String nombre;
    private String descripcion;

    public TipoLaboreo(String nombre, String descripcion) {
        this.nombre = nombre;
        this.descripcion = descripcion;
    }

    public String getNombre() {
        return nombre;
    }

    public String getDescripcion() {
        return descripcion;
    }

    public String mostrarTipoLaboreo() {
        return nombre;
    }

    public Boolean esSiembra() {
        return nombre.equals("Siembra");
    }

    public Boolean esCosecha() {
        return nombre.equals("Cosecha");
    }
}

File: python/laboreos/models/TipoLaboreo.py
from django.db import models


class TipoLaboreo(models.Model):
    nombre = models.CharField(max_length=100, primary_key=True)
    descripcion = models.TextField()
    
    class Meta:
        db_table = 'tipos_laboreo'
    
    def mostrar_tipo_laboreo(self):
        return self.nombre
    
    def es_siembra(self):
        return self.nombre == "Siembra"
    
    def es_cosecha(self):
        return self.nombre == "Cosecha"

